source/index.html.md

```diff
@@ -673,6 +673,32 @@ Exactly one payment method must be used.
     <div class="type">Optional</div>
   </dd>
 
+  <dt>
+    order_method
+    <span class="type">(<code>ecommerce</code>|<code>mail</code>|<code>telephone</code>)</span>
```

Comment:
@kse-clearhaus should we consider using this naming scheme in authorizer and klovedal instead of `mail_order` and `telephone_order`?

Reply:
IMO no, but that's what you're going to discuss on Friday.